docs/UsersGuide/source/Introduction.rst

```diff
@@ -1,177 +1,331 @@
 .. _Introduction:
 
-============
+==============
 Introduction
```

Comment:
The original intro was I basically a very detailed explanation of the components, which didn't seem remotely introductory. I copy-pasted the original info into the Components chapter so that we wouldn't lose data (and could expand upon it if that was desired) and then summarized components in the Intro. So there is overlap. I'd be fine with deleting the components page entirely or condensing the components part of the Intro even further. However, I don't think the current level of detail in the components section belongs in the introduction.